src/ui/components/Viewer.tsx

```diff
@@ -31,12 +31,10 @@ const Vertical = ({ toolboxLayout }: { toolboxLayout: ToolboxLayout }) => {
   useLayoutEffect(() => {
     const videoPanel = videoPanelRef.current;
     if (videoPanel) {
-      if (videoPanel.isCollapsed() !== videoPanelCollapsed) {
-        if (videoPanelCollapsed) {
-          videoPanel.collapse();
-        } else {
-          videoPanel.expand();
-        }
+      if (videoPanelCollapsed) {
+        videoPanel.collapse();
+      } else {
+        videoPanel.expand();
```

Comment:
Something looks odd about this change. Was the panel reporting an incorrect value for `isCollapsed` before?